src/main/java/com/google/sps/data/DatastoreManager.java

```diff
@@ -128,8 +130,22 @@ public static void updateRepresentativeTabList(long repId, List<Long> tabIdList)
         List<Long> tabIds = (ArrayList<Long>) repEntity.getProperty(Constants.REP_TABS); 
         if (tabIds == null) {
             tabIds = new ArrayList<>(); 
+            tabIds.addAll(tabIdList);
+        }
+        else{
+            List<String> tabNames = DatastoreEntityToObjectConverter.convertNamesFromTabs(tabIds);
+            for (Long tabId : tabIdList) {
+                Key tabEntityKey = KeyFactory.createKey(Constants.TAB_ENTITY_TYPE, tabId);
+                Entity tabEntity = (Entity) datastore.get(tabEntityKey); 
+                String name = (String) tabEntity.getProperty(Constants.TAB_NAME);
+                if (tabNames.contains(name)){
```

Comment:
make there is a space between anything and the following { 

here and everywhere else